test/src/test/resources/fixtures/replicate-bug.yml

```diff
@@ -0,0 +1,212 @@
+name: replicate bug
+enabled:
+  current: true
+  nextgen: false
+overallSchema:
+  #language=graphQL
```

Comment:
Idk about you, but this turns out really bad for me. You always get syntax errors saying you redefined a field or something rather